junebug/tests/test_api.py

```diff
@@ -57,14 +57,26 @@ def test_http_error(self):
 
     @inlineCallbacks
     def test_get_channel_list(self):
+        redis = yield self.get_redis()
+        config = self.default_channel_config
+
         resp = yield self.get('/channels')
-        yield self.assert_response(
-            resp, http.INTERNAL_SERVER_ERROR, 'generic error', {
-                'errors': [{
-                    'message': '',
-                    'type': 'NotImplementedError',
-                    }]
-                })
+        yield self.assert_response(resp, http.OK, 'channels listed', [])
+
+        yield Channel(redis, {}, config, u'test-channel-1').save()
+
+        resp = yield self.get('/channels')
+        yield self.assert_response(resp, http.OK, 'channels listed', [
+            u'test-channel-1',
```

Comment:
The docs at http://junebug.readthedocs.org/en/latest/http_api.html#get--channels don't specify what to return, but since real ids are uuids, it might be nicer to return some information about the channel in addition to the ids? So a list of dictionaries rather than just a list of ids. Thoughts?